client/ladder/ladder.tsx

```diff
@@ -183,35 +209,72 @@ export function Ladder({ matchmakingType: routeType }: LadderProps) {
     [setSearchQuery],
   )
 
+  useEffect(() => {
+    dispatch(
+      getMatchmakingSeasons({
+        onSuccess: () => setLastError(undefined),
+        onError: err => setLastError(err),
+      }),
+    )
+  }, [dispatch])
+
   useEffect(() => {
     const getRankingsAbortController = new AbortController()
     const searchRankingsAbortController = new AbortController()
+    const getPastRankingsAbortController = new AbortController()
```

Comment:
This again just seems very weird to me, why are we treating the current season as different from the historical ones?